src/pages/myGeotab/introduction.tsx

```diff
@@ -34,7 +36,7 @@ const importExport: ReactNode = (
             <li>Track Hours of Service (HOS) for regulations compliance</li>
         </ul>
         You can start using these tools right now by trying the <a target="_blank" rel="noreferrer" href="https://github.com/Geotab/sdk-dotnet-samples">.Net examples</a>,
-        {" "}<a href="../js-samples/">JavaScript examples</a> and <a target="_blank" rel="noreferrer" href="https://github.com/Geotab/sdk-java-samples">Java examples</a>.
+        {" " /*TODO: missing page*/}<a href="../js-samples/">JavaScript examples</a> and <a target="_blank" rel="noreferrer" href="https://github.com/Geotab/sdk-java-samples">Java examples</a>.
```

Comment:
This page is now created and in INTGR-123. There's also one for .NET, so might be better to link to that instead of GitHub.